lms/djangoapps/instructor/views/api_urls.py

```diff
@@ -31,6 +31,11 @@
         api_v2.InstructorTaskListView.as_view(),
         name='instructor_tasks'
     ),
+    re_path(
+        rf'^courses/{COURSE_ID_PATTERN}/change_due_date$',
+        api_v2.ChangeDueDateView.as_view(),
+        name='change_due_date_v2'
```

Comment:
You don't need the version in the name of the api endpoint since it's already namespaced under `instructor_api_v2`